src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurDetailLineGroupForCollector.java

```diff
@@ -146,8 +216,20 @@ protected String buildAccountingFieldsKey(ConcurStandardAccountingExtractDetailL
                 StringUtils.defaultIfBlank(detailLine.getOrgRefId(), StringUtils.EMPTY));
     }
 
-    protected String buildAccountingFieldsKeyForCashAdvance(
-            ConcurStandardAccountingExtractDetailLine detailLine, ConcurRequestedCashAdvance requestedCashAdvance) {
+    protected String buildAccountingFieldsKeyForAtmCashAdvance(ConcurStandardAccountingExtractDetailLine detailLine) {
+        return String.format(ACCOUNTING_FIELDS_KEY_FORMAT,

```

Comment:
There are a number of these building an accounting field key functions with similar logic.  I think this could be refactored to have one function do the string format, and the others call it.

Reply:
Ok, I'll refactor things so that the String.format() call is in a shared method.